draft-barnes-acme.md

```diff
@@ -1621,37 +1628,21 @@ token (required, string):
 least 128 bits of entropy, in order to prevent an attacker from guessing it.
 It MUST NOT contain any characters outside the URL-safe Base64 alphabet.
 
-~~~~~~~~~~
-{
-  "type": "dvsni",
-  "token": "evaGxfADs6pSRb2LAv9IZf17Dt3juxGJ-PCt92wr-oA"
-}
-~~~~~~~~~~
-
-In response to the challenge, the client uses its account private key to sign a
-JWS over a JSON object describing the challenge.  The validation object covered
-by the signature MUST have the following fields and no others:
-
-type (required, string):
-: The string "dvsni"
-
-token (required, string):
-: The token value from the server-provided challenge object
+authorizedKeys (required, string):
+: A serialized authorized keys object, base64-encoded.  This object MUST have
+only one entry, whose token value matches the "token" value in the challenge,
+and "key" value matches the client's account key.
 
 ~~~~~~~~~~
 {
   "type": "dvsni",
-  "token": "evaGxfADs6pSRb2LAv9IZf17Dt3juxGJ-PCt92wr-oA"
+  "token": "evaGxfADs6pSRb2LAv9IZf17Dt3juxGJ-PCt92wr-oA",
+  "authorizedKeys": "odyHtABZt47RZfacMq3zL...xIWRXBCCvl61bYo7ATU6Z4"
 }
 ~~~~~~~~~~
 
-The client serializes the validation object to UTF-8, then uses its account
-private key to sign a JWS with the serialized JSON object as its payload.  This
-JWS is NOT REQUIRED to have the "nonce" header parameter.
-
-The client will compute Z, the SHA-256 of the "signature" value from the JWS.
-The hash is calculated over the base64-encoded signature string.  Z is encoded
-in hexadecimal form.
+In response to the challenge, the client decodes the authorized keys object and
+computes its SHA-256 digest Z.  The client then encodes Z in hexadecimal form.
```

Comment:
What do you mean by decoding the authorized keys object? Just base-64 decode? How do you actually compute this digest? Do you mean `Z == SHA-256(B64-DECODE(authorizedKeys))`? This might be spelled out more clearly.

It's also probably worth mentioning that client should inspect the values and check that they match their Account Key etc.